pkg/config.go

```diff
@@ -39,6 +39,34 @@ func (*Config) Project() string {
 	return os.Getenv("GOOGLE_PROJECT")
 }
 
+// TlsVersion will be used current TLS Policy by this value.
+// The acceptable value MUST BE either`TLS_1_1`/`TLS_1_2`/`TLS_1_3`
+// Default is "TLS_1_2"
+func (*Config) TlsVersion() string {
+	tlsVersions := []string{"TLS_1_1", "TLS_1_2", "TLS_1_3"}
+	inputVersion := os.Getenv("MIN_TLS_VERSION")
+	for _, ver := range tlsVersions {
+		if inputVersion != ver || inputVersion == "" {
+			return "TLS_1_2"
+		}
+	}
+	return inputVersion
+}
```

Comment:
this function will always return `TLS_1_2` : https://play.golang.org/p/faw7I1c3T_R

you can change it to : 
```golang
func (*Config) TlsVersion() string {
	result := "TLS_1_2"
	tlsVersions := []string{"TLS_1_1", "TLS_1_2", "TLS_1_3"}
	for _, ver := range tlsVersions {
		if inputVersion == ver {
			result = ver
			break
		}
	}
	return result
}
```